guides/v2.2/rest/search-endpoint.md

```diff
@@ -0,0 +1,83 @@
+---
+group: rest-api
+title: Search with the /search endpoint
+contributor_name: 
+contributor_link: 
+---
+
+By using `GET {{URL}}rest/V1/search`, you can search through the catalog. The obtained results will come from your configured search engine (**Stores** > Settings > **Configuration** > **Catalog** > **Catalog Search** > **Search engine**).
```

Comment:
"Settings" also as **Settings** i expect?

Reply:
Actually, no. You don't click on that

Reply:
ah got it, right